geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/domain/DataCommandResult.java

```diff
@@ -240,6 +242,16 @@ public static DataCommandResult createRemoveResult(Object inputKey, Object value
     return result;
   }
 
+  public static DataCommandResult createClearResult(Throwable error, String errorString,
```

Comment:
This method is currently not being used anywhere.

Reply:
It was left in by mistake after making and then reverting some changes based on PR feedback. I'll remove this.